content/kotlin/concepts/math-methods/terms/abs/abs.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.abs()'
+Description: 'Returns the absolute value of the argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+## Description
+
+The `.abs()` method in Kotlin's Math library is used to calculate the absolute value of a numeric expression. The absolute value is the non-negative value of a number, effectively removing its negative sign, if present.
+
+### Syntax
+
+
+```pseudo
+Math.abs(n)
+```
+
+`n`: A numeric expression (e.g., `Int`, `Long`, `Double`, `Float`, etc.) for which you want to calculate the absolute value.
+
+The method takes the numeric expression n as its argument and returns the absolute value, ensuring that the result is always a non-negative value.
+
+### Example 
+
+```kotlin
+import kotlin.math.abs
+
+fun main() {
+    val x = -6.5
+    val y = 2
+    val z = -340.8
+
+    val absoluteX = abs(x)
+    val absoluteY = abs(y)
+    val absoluteZ = abs(z)
+
+    println("Absolute value of x: $absoluteX")
+    println("Absolute value of y: $absoluteY")
+    println("Absolute value of z: $absoluteZ")
+}
+```
+
+In this example, we import the abs function from the `kotlin.math` library and use it to calculate the absolute values of various numeric expressions. The results are then printed to the console.
+
+The output of this code will be:
+```shell
+Absolute value of x: 6.5
+Absolute value of y: 2
+Absolute value of z: 340.8
+```
+
+The .abs() method is particularly useful when you need to ensure that a value is treated as positive, regardless of its original sign.
```

Comment:
You can add a working codebyte after this. Try to keep the code different from above example.